### ISSUE TITLE
Shifted tiles/titles in high resolution

### ISSUE THREAD

**Issue by moberrauch** [Superjo149/auryo#243]
# 🐛 Bug report

## What issue are you facing
Opening Auryo with a high resolution (in my case 3840x2413 or 1920x1080) resultsin overlapping strings in the second row of tiles, in every menu with at least more than 2 rows of Tracks displayed. See the attached ![Screenshot](https://user-images.githubusercontent.com/36917865/75053082-f646fe80-54d0-11ea-99d2-56674fc4ce16.png).
Changing the width constantly changes the behaviour, e.g. tiles get shifted around to other places (which suggests the problem is whole tiles overlapping) and the number of erroneous tiles varies.
With a width <= 1561 everything seems to be fine.

## Steps to reproduce
Basically open it on fullscreen.

## Environment
[Sway](https://github.com/swaywm/sway) (wayland wm)
Auryo installed via AUR package `auryo-bin`

#### Which Operating System? 
ArchLinux with Kernel Linux 5.5.4-zen1-1-zen

#### Which version of Auryo? 
2.5.0 (update notes / commit on 2.5.1 don't suggest a fix for this anyway)

**Comment by moberrauch** [comment]
Sorry for bumping this issue again. I just installed 2.5.2 and while it seems fixed in the normal views (stream, likes, etc...) the search results are all scrambled up now. I did not test the search before, but it seams to be the same or a similar problem.
